src/main/kotlin/no/nav/klage/oppgave/api/controller/KlagebehandlingListController.kt

```diff
@@ -24,6 +24,8 @@ import java.util.*
 @RestController
 @Api(tags = ["kabal-api"])
 @ProtectedWithClaims(issuer = ISSUER_AAD)
+//TODO: Er det litt merkelig med "ansatte" på rot i path her?
```

Comment:
Jeg har sett mønstret der man starter med `/me`, og at det da utledes hvem det er i backend. Ved å beholde det som det er nå, så har vi en teoretisk mulighet å f.eks. la "admin" se på andre sine lister. Men det er absolutt ikke gjennomført. Vet heller ikke om vi har det behovet.

Man kan iaf argumentere for at alle kall som gjøres er på vegne av innlogget person. D.v.s. `me`. Man kan jo da droppe `me`, men da blir det ikke like eksplisitt.